com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKeyboard.cs

```diff
@@ -204,7 +224,13 @@ private void Awake()
             }
             Instance = this;
 
-             
+            if (dictationRecordIcon != null)
+            {
+                var material = new Material(dictationRecordIcon.material);
```

Comment:
please don't create a new material.   Impacts performance.

Why do we need this?

Reply:
If you are just changing the color, please use render.GetPropertyBlock/SetPropertyBlock  to change material settings...

it's cheaper

Reply:
removed